### DISCUSSION TITLE
/posts/React/React%20useState%20%ED%9B%85%EA%B3%BC%20%ED%81%B4%EB%A1%9C%EC%A0%80

### DISCUSSION THREAD

**Discussion by giscus** [phnml1/my-blog#15] [Announcements]
# /posts/React/React%20useState%20%ED%9B%85%EA%B3%BC%20%ED%81%B4%EB%A1%9C%EC%A0%80

자바스크립트의 어렵지만 중요한 클로저의 개념과 활용 이해해보자

http://localhost:3000/posts/Javascript/%ED%81%B4%EB%A1%9C%EC%A0%80%EC%97%90%20%EB%8C%80%ED%95%98%EC%97%AC

**Comment by phnml1** [comment]
ggg
